versions/2.0.0/asyncapi.md

```diff
@@ -92,6 +92,9 @@ A channel is an addressable component, made available by the server, for the org
 #### <a name="definitionsProtocol"></a>Protocol
 A protocol is the mechanism (wireline protocol OR API) by which [messages](#definitionsMessage) are exchanged between the application and the [channel](#definitionsChannel). Example protocol include, but are not limited to, AMQP, HTTP, JMS, Kafka, MQTT, STOMP, WebSocket.  
 
+#### <a name="definitionsBindings"></a>Bindings
+An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information that's necessary to utilize the API. Therefore, infrastructure details, deployment information, and everything that's not related to how to communicate with the API MUST be left apart.
```

Comment:
> An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information that's necessary to utilize the **API**. Therefore, infrastructure details, deployment information, and everything that's not related to how to communicate with the **API** MUST be left apart.

Not sure people will truly understand that the API in this case is protocol interface, I know that is what you mean, but for someone who are new to the protocols and the spec might confuse it🤔 Ofcourse they read it in the section above, but still. We could write it like so:

> An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, infrastructure details, deployment information, and everything that's not related to the protocol MUST be left apart.

Reply:
That's a great suggestion, Jonas. Changing it.